openlattice.yaml

```diff
@@ -1442,6 +1442,95 @@ paths:
               schema:
                 type: integer
             
+  ###################
+  ## DATA SET APIs ##
+  ###################
+  
+  /datastore/organization-database/{organizationId}/external-database-table:
+    get:
+      summary: Gets all OrganizationExternalDatabaseTable objects for an organization
+      operationId: getExternalDatabaseTables
+      tags: [dataset]
+      security:
+        - openlattice_auth: []
+        - http_auth: []
+      parameters:
+        - name: organizationId
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+      responses:
+        "200":
+          description: Success
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/OrganizationExternalDatabaseTable"
```

Comment:
I guess, this shouldn't be a relative reference based on this: https://github.com/openlattice/api/pull/386

Reply:
done